Environments/MS-VDI/orchestration.json

```diff
@@ -0,0 +1,386 @@
+{
+    "ModuleConfigurationsPath": "../../Modules",
+    "ModuleConfigurations": [
+        {
+            "Name": "VirtualNetworkHUB",
+            "ModuleDefinitionName": "VirtualNetwork",
+            "ResourceGroupName": "${Parameters.ModuleConfigurationParameters.VirtualNetworkHUB.ResourceGroup}",
+            "DependsOn": [    
+                "VirtualNetworkSPOKE"
+            ],
+            "Deployment": {
+                "OverrideParameters": {
+                    "vnetName": {
+                        "value": "${Parameters.ModuleConfigurationParameters.VirtualNetworkHUB.Name}"
+                    },
+                    "vnetAddressPrefixes": {
+                        "value": "${Parameters.ModuleConfigurationParameters.VirtualNetworkHUB.AddressPrefixes}"
+                    },
```

Comment:
Let's create a separate environment/orchestration for the HUB and SPOKE. Thinking is that the HUB and SPOKE should be independently deployed - decoupling it this way would be a better way to also prevent any VNET peering dependency issues. Also, we anticipate having multiple SPOKEs, so having it modularized this way would scale better.

Reply:
Completed. MS-VDI spoke uses Shared services HUB